float64[] joint_traj_goal
---
bool successed
string text